panel/io/server.py

```diff
@@ -131,15 +131,19 @@ def serve(panels, port=0, websocket_origin=None, loop=None, show=True,
       An HTML title for the application
     verbose: boolean (optional, default=True)
       Whether to print the address and port
+    location : boolean or panel.io.location.Location
+      Whether to create a Location component to observe and
+      set the URL location.
```

Comment:
Is there some legitimate reason not to create such a component?  If so, explain.